src/main/java/core/basesyntax/bookstore/BookstoreApplication.java

```diff
@@ -1,17 +1,15 @@
 package core.basesyntax.bookstore;
 
-import core.basesyntax.bookstore.service.BookService;
-import org.springframework.beans.factory.annotation.Autowired;
+import static org.springframework.data.web.config.EnableSpringDataWebSupport.PageSerializationMode.VIA_DTO;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.web.config.EnableSpringDataWebSupport;
 
 @SpringBootApplication
+@EnableSpringDataWebSupport(pageSerializationMode = VIA_DTO)
```

Comment:
Why do you need this annotation?

Reply:
![image](https://github.com/user-attachments/assets/8c24e22d-d91d-4c4b-b0c3-5a1c3c97b3aa)